test/utils/image/manifest.go

```diff
@@ -186,8 +185,8 @@ func ReplaceRegistryInImageURL(imageURL string) string {
 			break
 		}
 
-		log.Printf("Unknown Registry: %s\n", registryAndUser)
+		return "", fmt.Errorf("Registry: %s is missing in test/utils/image/manifest.go, please add the registry", registryAndUser)
```

Comment:
Let's add "add the registry, *otherwise the test will fail on air-gapped clusters*". Developers are likely to be more motivated to make that change when they know that there is a reason for it.

Reply:
Addeed